modules/openxBidAdapter.js

```diff
@@ -187,10 +203,25 @@ const OpenxAdapter = function OpenxAdapter() {
       }
     });
 
-    params.callback = 'window.$$PREBID_GLOBAL$$.oxARJResponse';
-    let queryString = buildQueryStringFromParams(params);
+    if (window.XMLHttpRequest && 'withCredentials' in new XMLHttpRequest()) {
```

Comment:
`withCredentials` should be available in any browser version that Prebid.js supports.  Are you seeing otherwise?